python/ray/tune/tests/test_tensorflow_trainable.py

```diff
@@ -71,6 +71,14 @@ def _train_check_global(config: Dict, checkpoint_dir: Optional[str] = None):
     tune.report(is_distributed=True)
 
 
+def _train_validate_session(config: Dict,
+                            checkpoint_dir: Optional[str] = None):
+    current_session = tune.session.get_session()
+    assert current_session is not None
+    assert current_session._trial_id != "default"
```

Comment:
can we test the user-facing API instead of directly accessing the private variable?

i.e., current_session.trial_id()?

Reply:
Good call out, fixed.